packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNavButton/useTeachingPopoverCarouselNavButton.tsx

```diff
@@ -42,6 +43,14 @@ export const useTeachingPopoverCarouselNavButton_unstable = (
     [onClick, setCurrentPage, index, onPageChange],
   );
 
+  const defaultTabProps = isSelected
```

Comment:
I think you want to use [`useTabsterAttribute()`](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-tabster/src/hooks/useTabsterAttributes.ts) here as it memoizes this behavior.